tests/manager/manager-executeBulkWrite_error-002.phpt

```diff
@@ -32,7 +32,7 @@ var_dump(iterator_to_array($cursor));
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-BulkWriteException: BulkWrite error :: E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 1 } :: E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 2 }
+BulkWriteException: Multiple write errors: "E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 1 }", "E11000 duplicate key error %s: phongo.manager_manager_executeBulkWrite_error_002%sdup key: { : 2 }"
```

Comment:
Static allocation? Seriously? :-/ PHP caps the message in warnings/errors to 1024 as well though... (not a problem for Exception messages though).

Do you think we can get them to change this value? Or increase it to 1024 or so?